Assets/ScriptableObject/Exo/Scripts/GenerateCards.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+public class GenerateCards : MonoBehaviour
+{
+    [SerializeField] private List<Card> _handOfCards;
+    [SerializeField] private int _numberOfCardsToDraw;
+    [SerializeField] private Deck _deck;
+    private void Start()
+    {
+        List<Card> copyDeck = new List<Card>(_deck.DeckOfCard);
+        for (int numberOfCard = 0; numberOfCard < _numberOfCardsToDraw; numberOfCard++)
```

Comment:
Le fait que tu nommes ta variable d'itération autrement que `i` est tout à ton honneur. Attention cependant à ce qu'elle ne ressemble pas trop à d'autres ou qu'on la confonde pour un usage qui ne serait pas le sien.
Un bon terme pour nommer les variables d'itération c'est `index` car c'est généralement pour parcourir un (ou plusieurs) tableau(x).
Dans ce cas-là : `cardIndex`

Reply:
C'est toi qui m'a dit de ne pas la nommer i...

Mais j'utiliserais index à l'avenir pour les variable d'itération

Reply:
Oui en effet, mais je t'ai pas dit de la renommer I. Je t'ai dit que le nom n'explicite pas trop son usage. On pourrait la confondre avec une autre variable qui ne sert pas à l'itération. C'est pour ça que "index" c'est un bon suffixe car on retrouve l'idée du "i" à l'intérieur.